### DISCUSSION TITLE
Housekeeping - Terceira semana

### DISCUSSION THREAD

**Discussion by ernestoresende** [thtsystems/open-visit#18] [Announcements]
Olá pessoal.

Na última sessão a gente começou a mexer com algumas coisas da API. A ideia é que tenhamos uma camada de API REST completamente documentada, aderindo a padrões de implementação estruturada sugeridas pelo spec do OpenAPI. Com os PR #17, nós demos alguns dos primeiros passos nessa direção.

O workspace `api` agora conta com a estrutura base necessária para construir as rotas. No README do projeto, é possível encontrar um link para a documentação da API no Postman. Para os interessados em contribuir com a construção da API ao longo das próximas semanas, esse é o seu chamado. As rotas que queremos construídas estão na documentação do Postman. Vocês podem escolher uma rota específica, ou um grupo de domínio, e começar a construir. As rotas já existentes devem dar uma ideia de como prosseguir.

> **Note**
> Não irei abrir issues individuais para cada rota. Caso um contribuidor queira iniciar o desenvolvimento de uma rota, ou grupo de rotas, sugiro que abra o issue e dê self-assign, pra que outros contribuidores saibam que você já está trabalhando naquela parte específica.

Por último, o documento [CONTRIBUTING.md](https://github.com/thtsystems/open-visit/blob/main/CONTRIBUTING.md) foi atualizado com mais instruções específicas sobre o setup do ambiente de desenvolvimento. Sugiro que leiam atentamente e tentem seguir os passos específicos.

Como dito na última sessão, vou deixar canais de comunicação mais acessíveis pra que possam tirar dúvidas de forma direta caso necessário:

- LinkedIn: https://www.linkedin.com/in/ernesto-resende/
- Discord: ernestoresende
- Twitter/X: https://twitter.com/ernestodotgg

**Comment by CalebeBirer** [comment]
hello can you help me with this error when import hono

calebebirer@calebebirer:~/Project_CBB$ npm create hono@latest my-app
Need to install the following packages:
  create-hono@latest
Ok to proceed? (y) y
internal/modules/cjs/loader.js:818
  throw err;
  ^
Error: Cannot find module 'node:http'
Require stack:
- /home/calebebirer/.npm/_npx/6bbb1d6f54609fb5/node_modules/create-hono/bin
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/calebebirer/.npm/_npx/6bbb1d6f54609fb5/node_modules/create-hono/bin:21719:33)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/calebebirer/.npm/_npx/6bbb1d6f54609fb5/node_modules/create-hono/bin'
  ]
}
npm ERR! code 1
npm ERR! path /home/calebebirer/Project_CBB
npm ERR! command failed
npm ERR! command sh -c create-hono "my-app"

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/calebebirer/.npm/_logs/2023-10-20T16_08_23_542Z-debug-0.log

**Reply by ernestoresende** [reply]
Moved this and answered in #43.
